test/labeling/test_analysis.py

```diff
@@ -24,6 +33,7 @@ def f(x):
 class TestAnalysis(unittest.TestCase):
     def setUp(self) -> None:
         self.lfa = LFAnalysis(np.array(L))
+        self.lfa_bis = LFAnalysis(np.array(L_2))
```

Comment:
What does `bis` refer to here?

Reply:
Agreed. It's a quite bad name. Sometimes, it means "2nd" in french... I replace this suffix by `_wo_abstain` for "without abstain".